Source/WebCore/rendering/svg/SVGResourcesCache.cpp

```diff
@@ -96,7 +96,41 @@ static inline bool rendererCanHaveResources(RenderObject& renderer)
     return renderer.node() && renderer.node()->isSVGElement() && !renderer.isSVGInlineText();
 }
 
-void SVGResourcesCache::clientStyleChanged(RenderElement& renderer, StyleDifference diff, const RenderStyle& newStyle)
+static bool hasStyleDifferencesAffectingResources(RenderObject& renderer, const RenderStyle* oldStyle, const RenderStyle& newStyle)
```

Comment:
Any specific reason for a static function rather than a lambda, at the point where it's used?

Reply:
No, I can change it to a lambda.